dali/operators/reader/loader/CMakeLists.txt

```diff
@@ -40,16 +40,23 @@ if (BUILD_NVDEC)
     "${CMAKE_CURRENT_SOURCE_DIR}/video_loader.cc")
 endif()
 
-set(DALI_OPERATOR_SRCS ${DALI_OPERATOR_SRCS} PARENT_SCOPE)
-
 set(DALI_OPERATOR_TEST_SRCS ${DALI_OPERATOR_TEST_SRCS}
   "${CMAKE_CURRENT_SOURCE_DIR}/loader_test.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/sequence_loader_test.cc"
   "${CMAKE_CURRENT_SOURCE_DIR}/numpy_loader_test.cc")
 
 if (BUILD_LIBSND)
   set(DALI_OPERATOR_TEST_SRCS ${DALI_OPERATOR_TEST_SRCS}
-    "${CMAKE_CURRENT_SOURCE_DIR}/nemo_asr_loader_test.cc")
+   "${CMAKE_CURRENT_SOURCE_DIR}/nemo_asr_loader_test.cc")
```

Comment:
?

Reply:
Removed a space by mistake, Imma put it back